sentry_streams/pyproject.toml

```diff
@@ -15,6 +15,11 @@ dependencies = [
     "pyyaml>=6.0.2",
     "jsonschema>=4.23.0",
     "sentry-kafka-schemas>=1.2.0",
+    "polars==1.30.0",
+    "pyarrow==19.0.0",
+    "pyarrow-stubs==19.0",
+    "pandas==2.2.3",
```

Comment:
Do you need pandas only for tests?
If yes please do not add this dependency. We should minimize the dependencies we import in a library. That makes it easier to use in other code bases.

Option 1: do without pandas 
Option 2: add it only to the dev section we use for tests.